example/lib/media_picker_icon_options.dart

```diff
@@ -0,0 +1,152 @@
+// SPDX-FileCopyrightText: 2022 Iconica
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+import 'package:example/media_picker_check.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_media_picker/flutter_media_picker.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+
+class MediaPickerPageIconOptions extends ConsumerStatefulWidget {
+  const MediaPickerPageIconOptions({required this.callback, Key? key})
+      : super(key: key);
+  final Function callback;
+
+  @override
+  ConsumerState<ConsumerStatefulWidget> createState() => _MediaPickerIconOptionsState();
+}
+
+class _MediaPickerIconOptionsState extends ConsumerState<MediaPickerPageIconOptions> {
+  @override
+  Widget build(BuildContext context) {
+    var mediaService = ref.read<MediaPickerService>(mediaPickerServiceProvider);
+    var audioService = ref.read<AudioService>(audioPlayerServiceProvider);
+    return Wrap(
+      children: [
+        Container(
+          width: MediaQuery.of(context).size.width,
+          decoration: const BoxDecoration(
+            color: Colors.white,
+            borderRadius: BorderRadius.only(
+              topLeft: Radius.circular(15),
+              topRight: Radius.circular(15),
+            ),
+          ),
+          child: Column(
+            children: [
+              const SizedBox(
+                height: 8,
+              ),
+              Container(
+                width: 70,
+                height: 5,
+                decoration: BoxDecoration(
+                  borderRadius: BorderRadius.circular(100),
+                  color: const Color(0xFF000000).withOpacity(0.50),
+                ),
+              ),
+              const SizedBox(
+                height: 14,
+              ),
+              const Text(
+                'Create/Pick',
+                style: TextStyle(
+                  fontWeight: FontWeight.w900,
+                  fontSize: 20,
+                ),
+              ),
+              const SizedBox(
+                height: 15,
+              ),
+              MediaPicker(
+                buttonType: ButtonType.icons,
+                horizontalSpacing: 40,
+                verticalSpacing: 20,
+                mediaPickerInputs: [
+                  MediaPickerInputPhoto(
+                    label: "Make photo",
```

Comment:
Use of single or double quotes is inconsistent. Please choose one of the two, I prefer single quotes.

Reply:
fixed